src/index.js

```diff
@@ -0,0 +1,20 @@
+// import _ from 'lodash';
+import './style.css';
+
+const input = document.getElementById('input');
+const addBtn = document.getElementById('addBtn');
+const list = document.getElementById('list');
+
+addBtn.addEventListener('click', () => {
+  const newList = document.createElement('li');
+  newList.innerHTML = input.value;
+  list.appendChild(newList);
+
+  newList.addEventListener('click', () => {
+    newList.style.textDecoration = 'line-through';
+  });
+
```

Comment:
- [ ] Kindly add a function that renders the tasks array on page load. It should populate the dom with the tasks in the array.